plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -422,10 +451,38 @@ class NewCreateServiceModalForm extends Component {
     ];
   }
 
+  /**
+   * This function filters the error list in order to keep only the
+   * errors that should be displayed to the UI.
+   *
+   * @returns {Array} - Returns an array of errors that passed the filter
+   */
+  getUnmutedErrors() {
```

Comment:
I would vote against `filterErrorsByPaths` because it goes towards the wrong direction. I think we should focus on properly processing the errors through the reducers first.

When we get rid of `CONSTANTLY_UNMUTED_ERRORS`, having a `getRootErrors` utility function might make sense, even though it's logic is quite minimal.